scripts/train.py

```diff
@@ -0,0 +1,15 @@
+from delphi.train.training import DDP,TrainingConfig, model_initialization, train_loop
+from delphi.train.utils import load_config
+from argparse import ArgumentParser
+
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument("--config", type=str, help="Path to the configuration file.")
```

Comment:
there's no config file, there's different arguments
```suggestion
```

Reply:
actually I'm fine with config file, I didn't see that before I commented on this in the call today